### PR TITLE
Add external chat and CoC

### PR DESCRIPTION
**Pull request by ahayzen-kdab**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official email address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+[INSERT CONTACT METHOD].
```
Comment (LeonMatthesKDAB) [inline CODE_OF_CONDUCT.md:64]: TODO: Insert contact method here.

I think listing your (@ahayzen-kdab ) and my zulip accounts is good enough?
Private message on zulip should have a reasonably low barrier.

**Review by LeonMatthesKDAB — CHANGES_REQUESTED** [review]
Still needs our contact info in case of CoC violation

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (ahayzen-kdab) [reply CODE_OF_CONDUCT.md:64]: ah yes good spot :-)

File: README.md [hunk 2]
```diff
@@ -14,13 +14,17 @@ SPDX-License-Identifier: MIT OR Apache-2.0
...
+[![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](CODE_OF_CONDUCT.md)
+
 | Crate | Status |
 | --- | --- |
 | cxx-qt | [![docs.rs](https://img.shields.io/docsrs/cxx-qt?logo=docsdotrs)](https://docs.rs/cxx-qt) [![Crates.io](https://img.shields.io/crates/v/cxx-qt)](https://crates.io/crates/cxx-qt) |
 | cxx-qt-build | [![docs.rs](https://img.shields.io/docsrs/cxx-qt-build?logo=docsdotrs)](https://docs.rs/cxx-qt-build) [![Crates.io](https://img.shields.io/crates/v/cxx-qt-build)](https://crates.io/crates/cxx-qt-build) |
 | cxx-qt-lib | [![docs.rs](https://img.shields.io/docsrs/cxx-qt-lib?logo=docsdotrs)](https://docs.rs/cxx-qt-lib) [![Crates.io](https://img.shields.io/crates/v/cxx-qt-lib)](https://crates.io/crates/cxx-qt-lib) |
 | qt-build-utils | [![docs.rs](https://img.shields.io/docsrs/qt-build-utils?logo=docsdotrs)](https://docs.rs/qt-build-utils) [![Crates.io](https://img.shields.io/crates/v/qt-build-utils)](https://crates.io/crates/qt-build-utils) |
 
+There is a Zulip chat available at [https://cxx-qt.zulipchat.com](https://cxx-qt.zulipchat.com).
```
Comment (LeonMatthesKDAB) [inline README.md:26]:
```suggestion
If you want to get in touch with us, feel free to join our Zulip Chat at:
[https://cxx-qt.zulipchat.com](https://cxx-qt.zulipchat.com).
There we openly discuss all things about CXX-Qt development.
```

I think that sounds a bit more welcoming :)

Comment (ahayzen-kdab) [reply README.md]: done :-)

**Review by LeonMatthesKDAB — APPROVED** [review]

**Pull request merged**